balancer-js/src/modules/swaps/__tests__/swaps.module.spec.ts

```diff
@@ -0,0 +1,105 @@
+import dotenv from 'dotenv';
```

Comment:
Could we remove the __tests__ folders and keep tests within the parent. For now we're just following the same convention as the front-end to keep consistency.